README.md

```diff
@@ -54,7 +54,7 @@ Here is a guide to how you might structure your documentation and presentation.
 
 - Neeti -
 - Anya -
-- Xiaozao -
+- Xiaozao - https://docs.google.com/document/d/1sZ_J3FM2iE9ZaFA6JRdLb_5t2hirzwK_BQgbHktw9MQ/edit?usp=sharing
```

Comment:
I recommend linking with a title rather than just the document URL. Also, I might suggest moving from a google doc format to GitHub markdown and having the write-up be a README.md file in your project subfolder! 

```suggestion
- Xiaozao - [Interactive Neural Networks - Documentation](https://docs.google.com/document/d/1sZ_J3FM2iE9ZaFA6JRdLb_5t2hirzwK_BQgbHktw9MQ/edit?usp=sharing)
```